packages/cli/package.json

```diff
@@ -30,6 +30,8 @@
     "@react-native-community/cli-types": "^3.0.0-alpha.2",
     "@types/mkdirp": "^0.5.2",
     "@types/semver": "^6.0.2",
+    "@types/ws": "^6.0.3",
```

Comment:
We use `ws@1`, shouldn't we match types? Ideally, we'd like to migrate to the newest version, but I'm not sure if we're able to do it without syncing Metro `ws` version as well

Reply:
It seems like the type definition for `ws@1` does not exist in DefinitelyTyped. The closest I could find was `3.0.0` or `0.0.42`
I'll look into this further to see if I can improve with typings.